packages/vuepress/vuepress-theme-titanium/styles/custom-blocks.styl

```diff
@@ -8,19 +8,19 @@
     border-left-style solid
     margin 1rem 0
   &.tip
-    background-color #f3f5f7
-    border-color #19AEEF
+    background-color #EDF2F7
```

Comment:
where are these colors coming from? I cannot seem to find matches in the style guide.